src/main/java/org/zalando/nakadi/service/EventTypeService.java

```diff
@@ -189,7 +189,15 @@ private void validateName(final String name, final EventTypeBase eventType) thro
 
     private void validateSchema(final EventTypeBase eventType) throws InvalidEventTypeException {
         try {
-            final JSONObject schemaAsJson = new JSONObject(eventType.getSchema().getSchema());
+            final String eventTypeSchema = eventType.getSchema().getSchema();
+            final JSONObject schemaAsJson = new JSONObject(eventTypeSchema);
+            final String parsedSchemaAsString = schemaAsJson.toString();
+
+            // bugfix ARUHA-563
+            // https://github.com/FasterXML/jackson-databind/issues/726
```

Comment:
How is this a jackson issue, if you use `org.json.JSONObject` to parse the schema?

Also, does this work if my schema is pretty-printed and contains whitespaces or newlines, and is it guaranteed to keep the order of properties?